src/main/java/hudson/plugins/dimensionsscm/PathUtils.java

```diff
@@ -51,7 +52,15 @@ static File getExecutable(String exeName) {
         return exe;
     }
 
-     static String normalizePath(String path) {
+    static String normalizePath(String path) {
         return path.contains("@") ? StringUtils.replace(path, "@", "@@") : path;
     }
+
+    static String normalizeSlashes(String path) {
+        String newPath = path.replaceAll("\\\\", "/");
```

Comment:
Silly change but using [`String#replace(char, char)`](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#replace-char-char-) would mean you didn't have to double up to 4 backslashes (just 2) and would probably be quicker because it doesn't use regular expressions.

Reply:
Yes, I will change it. Thanks!